static/app/views/starfish/views/spanSummaryPage/index.tsx

```diff
@@ -54,25 +48,25 @@ function SpanSummaryPage({params, location}: Props) {
               <BlockContainer>
                 <Block title={t('Operation')}>{span?.op}</Block>
                 <Block
-                  title={t('First Seen')}
-                  description={t(
-                    'The first time this span was ever seen in the current retention window'
-                  )}
+                  title={t('Throughput')}
+                  description={t('Throughput of this span per second')}
                 >
-                  <TimeSince date={spanMetrics?.['first_seen()']} />
-                  {firstSeenSpanEvent?.release && (
-                    <ReleasePreview release={firstSeenSpanEvent?.release} />
-                  )}
+                  {spanMetrics?.['spm()']?.toFixed(2)}/sec
```

Comment:
gotta convert that sp-minute to seconds! I will add backend support for this soon!

Reply:
Right! Thanks!